sdk/python/responsible-ai/text/responsibleaidashboard-text-classification-DBPedia.ipynb

```diff
@@ -85,7 +85,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "rai_example_version_string = \"20\""
+    "rai_example_version_string = \"21\""
```

Comment:
I think you need to bump up the model_name_suffix as well to get the notebook to pass builds